main.py

```diff
@@ -1,35 +1,60 @@
+import logging
 import sys
+from typing import List
 
-import inquirer
+from steamship.experimental.transports.chat import ChatMessage
 
 sys.path.insert(0, "src")
 from steamship import Steamship, SteamshipError
 from steamship.cli.ship_spinner import ship_spinner
 from termcolor import colored
 
 from api import MyAgent
-from src.utils import show_results, LoggingDisabled
 from tools import MyTool
 
 
+def show_results(response_messages: List[ChatMessage]):
+    print(colored("\nResults: ", "blue", attrs=["bold"]))
+    for message in response_messages:
+        if message.mime_type and message.mime_type.startswith("image"):
+            print(
+                f"LLM response ('{message.text}') contained an image: \n {message.url}",
+                end="\n\n",
+            )
+        else:
+            print(message.text, end="\n\n")
+
+
+class LoggingDisabled:
+    """Context manager that turns off logging within context."""
+
+    def __enter__(self):
+        logging.disable(logging.CRITICAL)
+
+    def __exit__(self, exit_type, exit_value, exit_traceback):
+        logging.disable(logging.NOTSET)
+
+
 def main():
     Steamship()
-    questions = [
-        inquirer.List(
-            "action",
-            message="Do you want to run your agent or your tool?",
-            choices=["Agent", "Tool"],
-            carousel=True,
-        ),
-    ]
-    answers = inquirer.prompt(questions)
+    # questions = [
```

Comment:
I think kind of nice to have the option to develop & run the tool on its own - is it causing problems to have the option to choose?